core/src/main/java/com/linecorp/armeria/client/retry/RetryStrategy.java

```diff
@@ -136,11 +133,6 @@ static RetryStrategy onStatus(
      * @param ctx the {@link ClientRequestContext} of this request
      * @param cause the {@link Throwable} which is raised while sending a request. {@code null} it there's no
      *              exception.
-     *
-     * @see RetryingClientBuilder#responseTimeoutMillisForEachAttempt(long)
-     *
-     * @see <a href="https://line.github.io/armeria/advanced-retry.html#per-attempt-timeout">Per-attempt
-     *      timeout</a>
      */
     CompletionStage<Backoff> shouldRetry(ClientRequestContext ctx, @Nullable Throwable cause);
```

Comment:
So we don't have mention about `Exceptions.peel()`?

Reply:
No, we don't. I found all the examples in `client-retry.rst` also use the exceptions as they are. I think it's really up to a user to use `Exceptions.peel()` or not.

Reply:
I added a tip about using `Exceptions.peel()` in `client-retry.rst` instead.